### PR TITLE
UAF-4265 : BUG - Invoice Status Missing from CAB AP Processing Screen.

### PR DESCRIPTION
**Pull request by jreeseUA**
UAF-4265 : BUG - Invoice Status Missing from CAB AP Processing Screen. Applied changes from 'edu\arizona\kfs\module\cab\businessobject\PurchasingAccountsPayableDocument.java' to 'org\kuali\kfs\module\cab\businessobject\PurchasingAccountsPayableDocument.java' and deleted: 'edu\arizona\kfs\module\cab\businessobject\PurchasingAccountsPayableDocument.java' and '/edu/arizona/kfs/module/cab/businessobject/datadictionary/PurchasingAccountsPayableDocument.xml'; because attempts to get Struts and PurApLineForm.java to use 'edu\arizona\kfs\module\cab\businessobject\PurchasingAccountsPayableDocument.java' and its Invoice Status using edu.arizona overrides required creation and modification of at least 17 edu.arizona override files and was spilling into Rice overrides.

### CODE REVIEW

File: kfs-cam/src/main/java/org/kuali/kfs/module/cab/businessobject/PurchasingAccountsPayableDocument.java [hunk 1]
```diff
@@ -25,25 +25,29 @@
...
 
 import org.apache.commons.lang.StringUtils;
-import org.kuali.kfs.module.cab.CabConstants;
 import org.kuali.kfs.module.cab.CabPropertyConstants;
-import org.kuali.kfs.module.purap.document.PaymentRequestDocument;
-import org.kuali.kfs.module.purap.document.VendorCreditMemoDocument;
 import org.kuali.kfs.sys.businessobject.FinancialSystemDocumentHeader;
 import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.kfs.sys.service.impl.KfsParameterConstants;
+import org.kuali.rice.coreservice.framework.parameter.ParameterConstants.NAMESPACE;
```
Comment (ghost) [inline kfs-cam/src/main/java/org/kuali/kfs/module/cab/businessobject/PurchasingAccountsPayableDocument.java:32]: Why is this needed?

File: kfs-cam/src/main/java/org/kuali/kfs/module/cab/businessobject/PurchasingAccountsPayableDocument.java [hunk 2]
```diff
@@ -25,25 +25,29 @@
...
 import org.kuali.rice.krad.util.ObjectUtils;
 
+import edu.arizona.kfs.module.cab.CabConstants;
+import edu.arizona.kfs.module.purap.document.PaymentRequestDocument;
+import edu.arizona.kfs.module.purap.document.VendorCreditMemoDocument;
+
 /**
  * @author Kuali Nervous System Team (kualidev@oncourse.iu.edu)
  */
+@NAMESPACE(namespace = KfsParameterConstants.PURCHASING_NAMESPACE)
```
Comment (ghost) [inline kfs-cam/src/main/java/org/kuali/kfs/module/cab/businessobject/PurchasingAccountsPayableDocument.java:47]: Why is this needed?

**Review by ghost — COMMENTED** [review]
Initial review complete. Please see my comments.

File: kfs-cam/src/main/java/org/kuali/kfs/module/cab/businessobject/PurchasingAccountsPayableDocument.java (hunk 2 above)
Comment (sskinner) [reply kfs-cam/src/main/java/org/kuali/kfs/module/cab/businessobject/PurchasingAccountsPayableDocument.java:47]: This is used as a new way of identifying namespace, as opposed to keeping it in a spring file. Without it, an STE pops up everytime the server starts while checking DD definitions and such.

Comment (ghost) [reply kfs-cam/src/main/java/org/kuali/kfs/module/cab/businessobject/PurchasingAccountsPayableDocument.java:47]: Let me be more specific: Why is this needed here? The delivered class did not have this namespace annotation.

Comment (jreeseUA) [reply kfs-cam/src/main/java/org/kuali/kfs/module/cab/businessobject/PurchasingAccountsPayableDocument.java:47]: The '@NAMESPACE(namespace = KfsParameterConstants.PURCHASING_NAMESPACE)' was part of the edu.az...PurchasingAccountsPayableDocument.java code from a previous  mod/bug so I pulled it over into the org.kuali copy when I moved its only method back into the org.kuali copy.

**Comment by ghost** [comment]
After Slack discussion, decided not to remove this annotation. Looks good!
